card-game-client/app/components/ui/admin-card.tsx

```diff
@@ -0,0 +1,166 @@
+import React, { useEffect, useState } from "react";
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableHeader,
+  TableRow,
+} from "./table";
+import Modal from "react-modal";
+import GameCard from "./game-card";
+import Image from "next/image";
+
+const categories = [
+  {
+    id: 1,
+    category: "Movies",
+    subcategory: "Harry Potter",
+    question: "Who played Harry Potter in the movies?",
+    image:
```

Comment:
don't know might be edit later?
 because this is just mock data

Reply:
yes, will remove while implementing api